### PR TITLE
Artifactory related changes

### LINKED ISSUE #23505: Artifactory updates
Replace artifacts below:
io.openliberty.org.glassfish.jaxb:codemodel:4.0.0-M4
io.openliberty.org.glassfish.jaxb:jaxb-core:4.0.0-M4
io.openliberty.org.glassfish.jaxb:jaxb-jxc:4.0.0-M4
io.openliberty.org.glassfish.jaxb:jaxb-runtime:4.0.0-M4
io.openliberty.org.glassfish.jaxb:jaxb-xjc:4.0.0-M4
io.openliberty.org.glassfish.jaxb:txw2:4.0.0-M4
io.openliberty.org.glassfish.jaxb:xsom:4.0.0-M4
org.glassfish.gmbal:gmbal-api-only:3.1.0-b001
with:
io.openliberty.org.glassfish.jaxb:codemodel:4.0.1
io.openliberty.org.glassfish.jaxb:jaxb-core:4.0.1
io.openliberty.org.glassfish.jaxb:jaxb-jxc:4.0.1
io.openliberty.org.glassfish.jaxb:jaxb-runtime:4.0.1
io.openliberty.org.glassfish.jaxb:jaxb-xjc:4.0.1
io.openliberty.org.glassfish.jaxb:txw2:4.0.1
io.openliberty.org.glassfish.jaxb:xsom:4.0.1
org.glassfish.gmbal:gmbal-api-only:4.0.3
in artifactory, bnd and files

### PR DESCRIPTION
**Pull request by BerksanAtes**
Fixes #23505

### CODE REVIEW

**Comment by BerksanAtes** [comment]
#libby-build

**Comment by BerksanAtes** [comment]
#libby-build

File: dev/io.openliberty.xmlws.4.0.internal_fat/build.gradle [hunk 1]
```diff
@@ -22,7 +22,7 @@ dependencies {
 				'org.jvnet.staxex:stax-ex:1.7.7',
 				'org.glassfish.jaxb:jaxb-runtime:3.0.+',
 				'org.glassfish.jaxb:jaxb-core:3.0.+',
-				'org.glassfish.gmbal:gmbal-api-only:3.1.0-b001'
+				'org.glassfish.gmbal:gmbal-api-only:3.+'
```
Comment (neuwerk) [inline dev/io.openliberty.xmlws.4.0.internal_fat/build.gradle]: This one says 3.+ but seems like in an earlier one you set: 'org.glassfish.gmbal:gmbal-api-only:4.0.3'

**Comment by BerksanAtes** [comment]
#build

**Review by neuwerk — DISMISSED** [review]

**Comment by BerksanAtes** [comment]
#build

**Review by neuwerk — DISMISSED** [review]
Perfect! Thanks for doing this!

**Comment by BerksanAtes** [comment]
#build

**Review by neuwerk — APPROVED** [review]

**Comment by BerksanAtes** [comment]
#run-libby-bot

**Pull request merged**